src/main/java/io/reactivex/Flowable.java

```diff
@@ -10345,6 +10345,17 @@ public final Disposable forEachWhile(final Predicate<? super T> onNext, final Co
      * is subscribed to. For this reason, in order to avoid memory leaks, you should not simply ignore those
      * {@code GroupedPublisher}s that do not concern you. Instead, you can signal to them that they may
      * discard their buffers by applying an operator like {@link #ignoreElements} to them.
+     * <p>
+     * By default, a {@link GroupedFlowable} buffers only {@code Flowable.BUFFER_SIZE} items to emit.
```

Comment:
The wording in these two paragraphs are odd and the terminology is slightly off. Could you please use this (revised) wording from the issue as the only paragraph instead:

```html
<p>
Note that the {@link GroupedFlowable}s should be subscribed to as soon as possible, otherwise, 
the unconsumed groups may starve other groups due to the internal backpressure
coordination of the {@code groupBy} operator. Such hangs can be usually avoided by using 
{@link #flatMap} or {@link #concatMapEager} and overriding the default maximum concurrency 
value to be greater or equal to the expected number of groups, possibly using 
{@link Integer.MAX_VALUE} if the number of expected groups is unknown.
```